### PR TITLE
RM-58005 Release over_react_codemod 1.3.2 (CPLAT-7563 comment escape hatch)

### PR DESCRIPTION
**Pull request by aaronlademann-wf**
This release updates the React 16 / Component2 executables to utilize the `Ignorable` class - ensuring that consumers can utilize a comment like so to fend off edge cases in which the codemod should not apply a patch.

```dart
// orcm_ignore
```

FYA: @greglittlefield-wf @joebingham-wk @kealjones-wk @sydneyjodon-wk

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/creator_utils.dart [hunk 1]
```diff
@@ -119,18 +121,21 @@ class DependencyCreator {
...
-  bool get asOverride => (gitOverride.isNotEmpty || pathOverride.isNotEmpty);
+  bool get asOverride =>
+      asNonGitOrPathOverride ||
+      gitOverride.isNotEmpty ||
+      pathOverride.isNotEmpty;
 
   @override
   String toString() {
-    if (asOverride) {
+    if (asOverride && !asNonGitOrPathOverride) {
```
Comment (kealjones-wk) [inline lib/src/creator_utils.dart]:
```suggestion
    if (isOverride) {
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+## [1.3.2](https://github.com/Workiva/over_react_codemod/compare/1.3.1...1.3.2)
+
+- Enable the use of `// orcm_ignore` comments when running the react16 / component2 codemods.
```
Comment (kealjones-wk) [inline CHANGELOG.md]:
```suggestion
- Enable the use of `// orcm_ignore` comments when running the react16 / component2 codemods.
- Update DependencyOverrideUpdater to allow for version overrides
```

File: lib/src/constants.dart [hunk 3]
```diff
@@ -104,7 +104,13 @@ final RegExp overReactDependencyRegExp = RegExp(
...
 /// Regex to find the dependency pubspec.yaml key.
 final RegExp dependencyRegExp = RegExp(
-  r'^dependencies:$',
+  r'^dependencies:\s*$',
+  multiLine: true,
+);
+
+/// Regex to find the dependency pubspec.yaml key.
+final RegExp devDependencyRegExp = RegExp(
+  r'^dev_dependencies:\s*$',
```
Comment (kealjones-wk) [inline lib/src/constants.dart]: test

**Comment by kealjones-wk** [comment]
+10

**Comment by kealjones-wk** [comment]
@Workiva/release-management-p

**Pull request merged**